### ISSUE TITLE
add assertions if table contains values

### ISSUE THREAD

**Issue by ChristianKuehnel** [bluebird75/luaunit#85]
Proposal for extension:
As a developer I want to check in my test case if a table does (not) contain a certain value.

It could look like this in the code:
```
table = some_function_to_be_tested(...)
luaunit.assertContains(table, "some value")
luaunit.assertNotContains(table, "other value")
```
This is my current workaround:
```
tablex = require('pl.tablex')
function contains(table, value)
  return tablex.find(table, value) ~= nil
end

luaunit.assertTrue(contains(table,"some value"))
luaunit.assertFalse(contains(table,"other value"))
```

**Comment by n1tehawk** [comment]
Hi Christian!

This implies that you do not know the corresponding table key, correct?
Otherwise you could do `luaunit.assertNotIsNil(t[key])`.

Regards, NiteHawk

**Comment by ChristianKuehnel** [comment]
Hi NiteHawk,

correct. I want to use this mainly for testing arrays and there I do not care about the keys or the order of the values in the array.

Regards, 
Christian

**Comment by bluebird75** [comment]
The request is sound, we'll try to work on something to address this need.

**Comment by bluebird75** [comment]
closed by #87
